WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Summary Section/Edit Order Status/OrderStatusListViewController.swift

```diff
@@ -126,6 +132,7 @@ extension OrderStatusListViewController {
             didSelectCancel?()
             return
         }
+        activateRightSpinner()
```

Comment:
I'm not sure if this is a good idea. It looks like this view controller was designed to be decoupled from the actual action that is performed, so I think it will make sense for the consumer on `didSelectApply` side to decide if this VC should render a spinner or not.